### DISCUSSION TITLE
Incorrect installation of PicoGK

### DISCUSSION THREAD

**Discussion by UgoW** [leap71/PicoGK#7] [Q&A] [answered]
Hi!
I'm having a little issue with the installation of PicoGK. I've successfully completed all the installation steps, but no colors are appearing in the examples. Is this a bug on my end, or is it normal?

**Comment by UgoW** [comment]
> Hi!
> I'm having a little issue with the installation of PicoGK. I've successfully completed all the installation steps, but no colors are appearing in the examples. Is this a bug on my end, or is it normal?
 Failed to load Light Setup - your viewer will look dark

with this error :
Could not find file '/Users/..../..../..../..../...../..../net7.0/PicoGKDefaultEnv.zip'.
 Failed to load Light Setup - your viewer will look dark

**Reply by LinKayser** [reply, marked as answer]
Make sure your subdirector/submodule is called “picogk” otherwise the viewer will not find the textures needed to create the image based lighting environment. If you want to use your custom paths you can pass the path of one of the light setups in the library.go command. The lighting environments can be found in the #ViewerEnvironment subfolder of PicoGK (https://github.com/leap71/PicoGK/tree/main/ViewerEnvironment)

Lin
